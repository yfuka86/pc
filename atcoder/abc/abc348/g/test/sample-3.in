6
9 7
2 4
7 1
-1000 0
3 4
8 5
