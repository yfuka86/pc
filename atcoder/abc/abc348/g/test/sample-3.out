6
10
17
20
22
-978
